Wandb run name:  usual-microwave-18
Detected classes and indices: {'person_eight': 0, 'person_five': 1, 'person_four': 2, 'person_nine': 3, 'person_one': 4, 'person_seven': 5, 'person_six': 6, 'person_ten': 7, 'person_three': 8, 'person_two': 9}
Traceback (most recent call last):
  File "c:\Users\midas\midas_model_experimental_umit\time_series\train.py", line 53, in <module>
    main()
  File "c:\Users\midas\midas_model_experimental_umit\time_series\train.py", line 47, in main
    train_loader, val_loader = get_dataloaders(config)
  File "c:\Users\midas\midas_model_experimental_umit\time_series\utilities.py", line 75, in get_dataloaders
    dataset = SeismicDataset(data_dir, window_size, window_stride)
  File "c:\Users\midas\midas_model_experimental_umit\time_series\utilities.py", line 38, in __init__
    data = pd.read_csv(file_path)
  File "C:\Users\midas\anaconda3\envs\num_model\lib\site-packages\pandas\io\parsers\readers.py", line 1026, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "C:\Users\midas\anaconda3\envs\num_model\lib\site-packages\pandas\io\parsers\readers.py", line 620, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "C:\Users\midas\anaconda3\envs\num_model\lib\site-packages\pandas\io\parsers\readers.py", line 1620, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "C:\Users\midas\anaconda3\envs\num_model\lib\site-packages\pandas\io\parsers\readers.py", line 1880, in _make_engine
    self.handles = get_handle(
  File "C:\Users\midas\anaconda3\envs\num_model\lib\site-packages\pandas\io\common.py", line 873, in get_handle
    handle = open(
  File "C:\Users\midas\anaconda3\envs\num_model\lib\codecs.py", line 309, in __init__
    def __init__(self, errors='strict'):
KeyboardInterrupt
